.github/workflows/CI.yml

```
@@ -12,13 +12,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [stable, 1.39.0]
+        rust: [stable, 1.44.0]
```

As per https://github.com/tokio-rs/tracing/pull/934#issuecomment-675197598, this should be
```suggestion
        rust: [stable, 1.40.0]
```